dockstore-webservice/src/main/java/io/dockstore/webservice/core/metrics/ExecutionStatusCountMetric.java

```diff
@@ -62,17 +63,23 @@ public class ExecutionStatusCountMetric extends CountMetric<ExecutionStatusCount
     private Map<ExecutionStatus, Integer> count = new EnumMap<>(ExecutionStatus.class);
 
     @Column(nullable = false)
-    @Schema(description = "Number of successful executions", requiredMode = RequiredMode.REQUIRED)
+    @Schema(description = "Number of successful executions", accessMode = AccessMode.READ_ONLY, example = "5")
     private int numberOfSuccessfulExecutions;
 
     @Column(nullable = false)
-    @Schema(description = "Number of failed executions. An execution may have failed because it was semantically or runtime invalid", requiredMode = RequiredMode.REQUIRED)
+    @Schema(description = "Number of failed executions. An execution may have failed because it was semantically or runtime invalid", accessMode = AccessMode.READ_ONLY, example = "2")
     private int numberOfFailedExecutions;
 
+    @Column(nullable = false)
+    @Schema(description = "Number of aborted executions. An execution is aborted if it's execution is stopped after it has started", accessMode = AccessMode.READ_ONLY, example = "0")
+    private int numberOfAbortedExecutions;
```

Comment:
Hmm, I lean towards keeping it as is because like you said, you could argue that about the other statuses as well. How do others feel about this? @denis-yuen @svonworl

Reply:
Oooo toughie.  One less "nullable" is one less special case...  I'd don't have any deeper recommendations, lol...

Reply:
Seems kinda zen. Like if a platform doesn't track aborted runs and doesn't report them either, did they really occur? :laughing: 

I'd leave it, I think it's obvious that we don't claim to represent everything that ever occured, just what we have access to.